README.md

```diff
@@ -4,14 +4,15 @@ This is the source code of the <https://qiblih.com/> website.
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/2e83bf87-458c-4e0c-bb8d-130feb12a85b/deploy-status)](https://app.netlify.com/sites/qiblih/deploys)
 
-The `public` folder gets deployed to the <https://www.netlify.com/> web host.  
```

Comment:
Ah, interesting, didn't know that. I've switched to `<br>` so as to avoid IDE's such as mine being able to auto-strip trailing whitespace from doing so, if that's ok.